content/en/docs/concepts/security/security-checklist.md

```diff
@@ -0,0 +1,140 @@
+- Pods with secrets should have these automatically mounted through volumes.
+  This would allow adding secrets into the pod as environment variables or
+  files on the filesystem. This should be done preferentially as compared to
+  providing the pods service account RBAC access to the secret.
+- Secret resources stored within etcd should be [encrypted at
+  rest](/docs/tasks/administer-cluster/encrypt-data/).
+- Service accounts should not be mounted into pods that do not require them.
+  This can be configured by setting
+  [`automountServiceAccountToken`](/docs/tasks/configure-pod-container/configure-service-account/#use-the-default-service-account-to-access-the-api-server)
+  to `false` either within the service account to apply throughout the
+  namespace or specifically for a pod.
+- Kubernetes v1.22 and above - Utilise [Bound Service
+  Accounts](/docs/reference/access-authn-authz/service-accounts-admin/#bound-service-account-token-volume)
+  for time-bound service account credentials.
+
+## Container & Image scanning
+
+Scanning a container image can prevent critical vulnerabilities from being
+deployed to the cluster alongside the container image. Image scanning should be
+completed before deploying a container image to a cluster and is usually done
+as part of the deployment process in a CI/CD pipeline. The purpose of an image
+scan is to obtain information about possible vulnerabilities and their
+prevention in the container image, such as a [Common Vulnerability Scoring
+System
+(CVSS)](https://en.wikipedia.org/wiki/Common_Vulnerability_Scoring_System)
+score. If the result of the image scans is combined with the pipeline
+compliance rules, only properly patched container images will end up in
+Production.
+
+The blog post "[A Closter Look at NSA/CISA Kubernetes Hardening
+Guidance](/blog/2021/10/05/nsa-cisa-kubernetes-hardening-guidance/#building-secure-container-images)"
+provides more guidance on how to build secure container images.
+
+## Enabling seccomp
+
+Seccomp can improve the security of your workloads by reducing the Linux kernel
+syscall attack surface available inside containers. The seccomp filter mode
+leverages BPF to create allow or deny list of specific syscalls, named
+profiles. Those seccomp profiles can be enabled on individual workloads, [a
+security tutorial is available](/docs/tutorials/security/seccomp/). In
+addition, the [Kubernetes Security Profiles
+Operator](https://github.com/kubernetes-sigs/security-profiles-operator) is a
+project to facilitate the management and use of seccomp in clusters.
+
+Please note that Docker has been using [a default seccomp
+profile](https://docs.docker.com/engine/security/seccomp/) to only allow a
+restricted set of syscalls but Kubernetes is still not confining workloads by
+default. Fortunately, [Seccomp Default](/blog/2021/08/25/seccomp-default/), a
+new alpha feature to use a default seccomp profile for all workloads can now be
+enabled and tested.
```

Comment:
I think we could merge that into Pod Security?

Reply:
It is not really part of the Security Context, but it definitely would fit in the "Pod Security" section. 
Seccomp on the other hand is now part of the Security Context. So either it could be added to the Security Context section or placed directly below it.